src/test/prediction_service_test.cpp

```diff
@@ -74,14 +83,41 @@ ovms::Status getOutput(const KFSResponse& response, const std::string& name, KFS
     return StatusCode::INVALID_MISSING_INPUT;
 }
 
-ovms::Status getOutput(const TFSResponseType& response, const std::string& name, TFSOutputTensorIteratorType& it, size_t& bufferId) {
+static ovms::Status getOutput(const TFSResponseType& response, const std::string& name, TFSOutputTensorIteratorType& it, size_t& bufferId) {
     it = response.outputs().find(name);
     if (it != response.outputs().end()) {
         return StatusCode::OK;
     }
     return StatusCode::INVALID_MISSING_INPUT;
 }
 
+using inputs_info_elem_t = std::pair<std::string, std::tuple<ovms::shape_t, ovms::Precision>>;
+size_t calculateByteSize(const inputs_info_elem_t& e) {
+    auto& [inputName, shapeDatatypeTuple] = e;
+    auto& [shape, precision] = shapeDatatypeTuple;
+    size_t shapeProduct = std::accumulate(shape.begin(), shape.end(), 1, std::multiplies<size_t>());
```

Comment:
We could use that in all ovms src/test src places where we iterate over shape vector to accumulate